src/outputs/outputs.hpp

```diff
@@ -126,7 +128,7 @@ class OutputType {
 class HistoryOutput : public OutputType {
  public:
   explicit HistoryOutput(OutputParameters oparams) : OutputType(oparams) {}
-  void WriteOutputFile(Mesh *pm, ParameterInput *pin, bool flag) override;
+  void WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm) override;
```

Comment:
I would also recommend changing all `override` keywords to `final` for all the children of OutputType.

Reply:
not here, not now, please.